flate/stateless.go

```diff
@@ -59,7 +61,7 @@ var bitWriterPool = sync.Pool{
 
 // StatelessDeflate allows to compress directly to a Writer without retaining state.
 // When returning everything will be flushed.
-func StatelessDeflate(out io.Writer, in []byte, eof bool) error {
+func StatelessDeflate(out io.Writer, in []byte, eof bool, dict ...byte) error {
```

Comment:
I think it's best the dict is []byte instead of a variadic. Can't imagine people would be passing in anything other than a byte slice.

Reply:
Ahh I saw your latest comment.

Reply:
This will not break the API. Adding a `[]byte` will.

Reply:
Given this is a relatively recent addition and not very widely used likely, would you be open to just making it []byte instead?

Reply:
I'll consider it.